auth/presentation/src/main/java/com/gkp/auth/presentation/login/LoginEvents.kt

```diff
@@ -0,0 +1,7 @@
+package com.gkp.auth.presentation.login
+
+interface LoginEvents {
+    data object LoginEventsSuccess : LoginEvents
+    data object LoginEventsLoading : LoginEvents
```

Comment:
Whether something is loading or not is a state rather than a one-time event, no?